Makefile

```diff
@@ -218,4 +218,6 @@ failpoint-disable:
 	# Restoring failpoints...
 	@$(FAILPOINT_DISABLE)
 
-.PHONY: all ci vendor clean-test tidy
+clean: clean-test failpoint-disable deadlock-disable
```

Comment:
how about also cleaning the built artifacts in `bin` directory

Reply:
Good idea!

Reply:
Done.